### DISCUSSION TITLE
How to define a custom logo?

### DISCUSSION THREAD

**Discussion by cagix** [LiaScript/LiaScript#154] [Q&A] [answered]
Is there a way to define a custom icon to replace the origami pelican, which is shown at the top of every book? The logo option in the pseudo YAML (configuration comment) seems only to relate to the logo for the PWA in "home" view.

**Comment by SebastianZug** [comment, marked as answer]
Take a view to [LiaScriptLiveEditor](https://liascript.github.io/LiveEditor/?/show/code/H4sIAAAAAAAAA4WOMU8DMQyF9/wKU+ZLJMSAKsTQoVIlBgaYe7nEzVlN4ijJ9ei/JznEjAfLfn6f/F4fhkEIMhz30GquNZW9UkvyrK1c6UoBLWnJ2am+pb4pwyFwLKrOS5iUVRbVOzs+f36dD5idvmqLgfB8zEhTE2S5OfX89JK+h399MkUnhBiGNyEe4RQt3cgu6D1CZ4sQhzsUrJWig7EnH2FFKCtVM6OFOiNQ0A7BYiJTm6QL+IYCxe1qMFbMwBFQmxl2xZPFnRTiI/OkJ39vxt+v2sPYyRGSzjpgx6i0cxuwbAkunOHOSwbTWsEClVnCqULGSzNF06W/SAXKzGvsr7cgGwIeb+il+AEgx0zAiwEAAA== ). The example shows how to reference my university :-)

Best wishes
Sebastian

**Reply by cagix** [reply]
> Take a view to [LiaScriptLiveEditor](https://liascript.github.io/LiveEditor/?/show/code/H4sIAAAAAAAAA4WOMU8DMQyF9/wKU+ZLJMSAKsTQoVIlBgaYe7nEzVlN4ijJ9ei/JznEjAfLfn6f/F4fhkEIMhz30GquNZW9UkvyrK1c6UoBLWnJ2am+pb4pwyFwLKrOS5iUVRbVOzs+f36dD5idvmqLgfB8zEhTE2S5OfX89JK+h399MkUnhBiGNyEe4RQt3cgu6D1CZ4sQhzsUrJWig7EnH2FFKCtVM6OFOiNQ0A7BYiJTm6QL+IYCxe1qMFbMwBFQmxl2xZPFnRTiI/OkJ39vxt+v2sPYyRGSzjpgx6i0cxuwbAkunOHOSwbTWsEClVnCqULGSzNF06W/SAXKzGvsr7cgGwIeb+il+AEgx0zAiwEAAA== ). The example shows how to reference my university :-)
> 
> 
> 
> Best wishes
> 
> Sebastian 

thanks for your swift response! i've only seen `logo`, but `icon` makes absolutely sense 👍
